checklists/avd_checklist.en.json

```diff
@@ -518,64 +302,496 @@
             "guid": "2289b3d6-b57c-4fc6-9546-1e1a3e3453a3",
             "severity": "Low",
             "link": "https://docs.microsoft.com/azure/storage/files/storage-files-identity-ad-ds-enable"
         },
         {
             "category": "Identity",
-            "subcategory": "Requirements",
-            "text": "An Azure Active Directory tenant must be available with at least one subscription linked",
-            "description": "An Azure subscription must be parented to the same Azure AD tenant, that contains a virtual network that either contains or is connected to the Windows Server Active Directory or Azure AD DS instance.",
-            "guid": "6ceb5443-5125-4922-9442-93bb628537a5",
+            "subcategory": "Active Directory",
+            "text": "A Windows Server Active Directory forest/domain must be in sync with Azure Active Directory",
+            "description": "You can configure this using Azure AD Connect (for hybrid organizations) or Azure AD Domain Services (for hybrid or cloud organizations).",
+            "guid": "5119bf8e-8f58-4542-a7d9-cec166cd072a",
             "severity": "High",
-            "link": "https://docs.microsoft.com/azure/virtual-desktop/overview"
+            "link": "https://learn.microsoft.com/en-us/azure/virtual-desktop/prerequisites?tabs=portal#identity"
         },
         {
             "category": "Identity",
-            "subcategory": "Requirements",
-            "text": "A Windows Server Active Directory forest/domain is in sync with Azure Active Directory",
-            "description": "You can configure this using Azure AD Connect (for hybrid organizations) or Azure AD Domain Services (for hybrid or cloud organizations).",
-            "guid": "5119bf8e-8f58-4542-a7d9-cec166cd072a",
+            "subcategory": "Azure AD",
+            "text": "Configure Azure Files share for Azure Active Directory (Azure AD) Kerberos authentication",
+            "description": "If Azure Files is used and pre-requisites can be satisfied, it is recommended to configure (Azure AD) Kerberos authentication. This configuration will allow to store FSLogix profiles that can be accessed by hybrid user identities from Azure AD-joined or Hybrid Azure AD-joined session hosts without requiring network line-of-sight to domain controllers.",
+            "guid": "e777fd5e-c5f1-4d6e-8fa9-fc210b88e338",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-hybrid-identities-enable"
+        },
+        {
+            "category": "Compute",
+            "subcategory": "Golden Images",
+            "text": "Determine how applications will be deployed in AVD Host Pools",
+            "description": "Applications can be preinstalled in the golden image/s, can be attached using MSIX & AppAttach feature or distributing to hosts after pool deployment using traditional SW distribution methods.",
```

Comment:
Suggest to change "distributing" to "distributed" in description text.

Perhaps also add " such as Intune / MECM" at the end as an example software distribution method.